source/states/PlayCollisionState.hx

```diff
@@ -17,30 +18,46 @@ class PlayCollisionState extends FlxState
 	var test:FlxSprite;
 	var test2:FlxSprite;
 	var align:FlxSprite;
+
+	var projectiles:FlxGroup;
+	var enemies:FlxGroup;
+	var characters:FlxGroup;
+	var towers:FlxGroup;
```

Comment:
Thought here is that this game will revolve around a couple key groups of things. So just have these at the top level for ease of collisions and handling.

Reply:
Good idea